env/env.go

```diff
@@ -1,25 +1,40 @@
 package env
 
 import (
-	"github.com/EVODelavega/goq/aws"
+	"os"
+
+	goqaws "github.com/EVODelavega/goq/aws"
```

Comment:
aws specific imports in a general "env" package? That's not good. Just create a config package that is AWS specific with a `Get` func.